### ISSUE TITLE
Misnomer in FAQ.

### ISSUE THREAD

**Issue by Aaronepower** [rust-lang/rust-www#262]
In the [FAQ](https://github.com/rust-lang/rust-www/blob/master/faq.md#what-is-the-difference-between-consuming-and-moving) there is a section with the following heading, and body text. However to my knowledge passing by value doesn't mean the same thing as the rest. Passing by value is equivalent of the `Copy` trait, where the original owner can still use the data after it has passed the value to a different owner.
***
### What is the difference between passing by value, consuming, moving, and transferring ownership?

These are different terms for the same thing. In all cases, it means the value has been moved to another owner, and moved out of the possession of the original owner, who can no longer use it.
***

**Comment by steveklabnik** [comment]
We tend to use them in the same way in Rust-world, regardless if things are Copy or not. Other languages don't have the ownership rules, so they don't do the kind of invalidation we do.

**Comment by Aaronepower** [comment]
Yes, but it is an FAQ for people who aren't familiar with Rust. There should be some clarity that the term is being used in a different way than someone coming from a different language might expect. This could be clarified in the section following which does discuss the `Copy` trait.

**Comment by AndrewBrinker** [comment]
I think you're right. It's probably worthwhile to mention the `Copy` trait and state that a move does occur if the passed-in type implements it, and make it clear that both the move and copy versions are considered "pass by value" (the notion of whether ownership was moved or copied is opaque from the perspective of the function).
